hook-emulator/transaction-manager.js

```diff
@@ -291,18 +325,16 @@ class TransactionManager {
             let completed = false;
 
             // Set the transaction process timeout and reject the promise if reached.
-            const failTimeout = setTimeout(() => {
+            const failTimeout = setTimeout(async () => {
```

Comment:
Do we need the async keyword since there is no asynchronous logic in the function body?